docs/DynamicESDL/DynamicESDL_Includer.xml

```diff
@@ -129,22 +129,35 @@
 
           <para>At this point you can test the query using WsECL.</para>
 
-          <para />
+          <para></para>
         </listitem>
 
         <listitem>
           <para>Publish the Dynamic ESDL definition.</para>
 
-          <para />
+          <para></para>
         </listitem>
 
         <listitem>
           <para>Bind the service methods to the Roxie queries using a
           configuration XML file.</para>
 
-          <para />
+          <para></para>
         </listitem>
       </itemizedlist>
+
+      <para><emphasis role="bold">DESDL and LDAP Security</emphasis></para>
+
+      <para>If your HPCC platform is configured to use for LDAP security, you
```

Comment:
"configured to use for LDAP" doesn't make sense.
Also, keep in mind that in the future we won't be 100% reliant on LDAP for authorization